packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -744,9 +748,13 @@ export interface SdkBasicServiceMethod<TServiceOperation extends SdkServiceOpera
 
 interface SdkPagingServiceMethodOptions {
   __raw_paged_metadata?: PagedResultMetadata;
+  /**
+   * @deprecated Use `nextLinkSegments` instead.
+   */
   nextLinkPath?: string;
+  nextLinkSegments?: SdkModelPropertyType[];
   nextLinkOperation?: SdkServiceOperation;
-  continuationTokenParameter?: SdkMethodParameter;
+  continuationTokenParameterSegments?: SdkModelPropertyType[];
```

Comment:
just curous why not `continuationTokenResponseSegments?: SdkModelPropertyType[] | SdkServiceResponseHeader[]`?

Reply:
Hah I notice the change for line 651.

Reply:
resolved due to https://github.com/Azure/typespec-azure/pull/2206/files#r1955718912

Reply:
already changed.